schema/spacewalk/common/tables/rhnVirtualInstance.sql

```diff
@@ -28,6 +28,7 @@ CREATE TABLE rhnVirtualInstance
     uuid               VARCHAR2(128),
     confirmed          NUMBER(1)
                            DEFAULT (1) NOT NULL,
+    payg               BOOLEAN,
```

Comment:
Hmm, as for now we never used "BOOLEAN" in the DB. We always used `CHAR(1)` with 'Y' and 'N' value.
This was for oracle compatibility which is not needed anymore. But we use it everywhere. I would like to keep it for now and make an official discussion about what we want to do in future.
And for backporting it is needed anyway.